src/SFA.DAS.Campaign.Web/Views/Shared/Components/_Navigation.cshtml

```diff
@@ -134,7 +134,7 @@
         <li class="navigation__list-item navigation__list-item--with-button">
             @*<partial name="./components/Form/_button" model="@{new ButtonViewModel(ButtonType.Primary){Shadow = true, Sparks = true, Text = "Register Interest"}}" />*@
 
-            @await Component.InvokeAsync("Button", new { type = ButtonType.Input, style = ButtonStyle.Primary, text = "Register Interest" })
+            @await Component.InvokeAsync("Button", new { type = ButtonType.Anchor, style = ButtonStyle.Primary, text = "Register Interest", href = "#modal-RegisterInterest", shadow = true, sparks = true })
```

Comment:
I'm not one who's best to comment on front end - however - this doesn't seem ideal to me, I'd personally prefer a much cleaner separation of front end to code to back end, I'm unclear as to what we are gaining by doing this.